TypeScriptReact.sublime-settings

```diff
@@ -0,0 +1,4 @@
+{
+    "auto_complete_triggers" : [ {"selector": "source.tsx", "characters": "."} ],
```

Comment:
It doesn't work. I wasted some hours trying to make it work and eventually found the reason [here](http://sublime-text-unofficial-documentation.readthedocs.org/en/latest/customization/settings.html):

>file type settings need to carry the name of the .tmLanguage syntax definition for the file type. Thus, for the .py file type, whose syntax definition is contained in Python.tmLanguage, the corresponding settings files would be called Python.sublime-settings.